src/EditorFeatures/Core.Wpf/SignatureHelp/Presentation/Signature.cs

```diff
@@ -216,8 +216,10 @@ private static IList<TaggedText> AddOptionalBrackets(bool isOptional, IList<Tagg
         {
             if (isOptional)
             {
-                var result = new List<TaggedText>();
-                result.Add(new TaggedText(TextTags.Punctuation, "["));
+                var result = new List<TaggedText>
```

Comment:
maybe

                List<TaggedText> result =
                [
                    new TaggedText(TextTags.Punctuation, "["),
                    .. list,
                    new TaggedText(TextTags.Punctuation, "]"),
                ];

Reply:
talked offline.  the nalayzer/fixer here does not want to remove var.  in c# 13 we're looking at expanding collection exprs to work here with `var` as well, so when that comes around, we'll likely want to update this to just work.

Reply:
Talked offline, and sounds like the analyzer purposefully doesn't change var semantics. Cyrus indicated there might be a C# 13 feature which magically makes this work with var.